### PR TITLE
FS-005: add cypress tests.

### PR DESCRIPTION
**Pull request by MykhailoForge**

### CODE REVIEW

File: cypress/support/commands.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+/// <reference types="cypress" />
+
+Cypress.Commands.add("getBySel", (testId) => {
```
Comment (stanislavSoro) [inline cypress/support/commands.ts:3]: Please change naming to getByTestId. getBySel is not clear. What Sel presents?

File: cypress/e2e/StackVisualizer.cy.ts [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+const inputTextAddNewStackItem = () => {
+  cy.getBySel(`data-list-text-input`).type(STACK_TEST_MSG);
+  cy.getBySel(`data-list-add-button`).click();
+  cy.getBySel(`data-list-items-container`).children().should("have.length", 4);
+  cy.contains(STACK_TEST_MSG);
+};
+
+const removeStackItem = () => {
+  cy.getBySel(`data-list-remove-button`).click();
+  cy.getBySel(`data-list-items-container`).children().should("have.length", 2);
```
Comment (stanislavSoro) [inline cypress/e2e/StackVisualizer.cy.ts:17]: How you can be sure that your Item is removed? Also if we change markup and there will be 2 divs this test will pass as well.

File: cypress/e2e/StackVisualizer.cy.ts [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+const STACK_TEST_MSG = "Test item";
+const PATH_NAME_STACK = "/stack";
+const STACK_LANG_MATCHER =
+  uaLocalization.translation.stackVisualizer.buttons.add;
+
+const inputTextAddNewStackItem = () => {
+  cy.getBySel(`data-list-text-input`).type(STACK_TEST_MSG);
+  cy.getBySel(`data-list-add-button`).click();
+  cy.getBySel(`data-list-items-container`).children().should("have.length", 4);
```
Comment (stanislavSoro) [inline cypress/e2e/StackVisualizer.cy.ts:11]: I don't like idea of checking children count. You are adding element. You need to check that it's presented in the list.

File: cypress/support/commands.ts (hunk 1 above)
Comment (MykhailoForge) [reply cypress/support/commands.ts:3]: Ok. Get by Sel (selector) is a ref to 
image

Comment (stanislavSoro) [reply cypress/support/commands.ts:3]: Sure, But in this example it references to a Selector. Project can feature several custom selectors, so it's better to stick to the context. In this case we have more flexibility.

Comment (MykhailoForge) [reply cypress/support/commands.ts:3]: Yeah, sure, obvious naming issue. I just refered to the doc in this moment
.

File: cypress/e2e/StackVisualizer.cy.ts (hunk 2 above)
Comment (MykhailoForge) [reply cypress/e2e/StackVisualizer.cy.ts:17]: before every test issue we make cy.visit(${path_name}), so i always sure that the starting number of items in tessted list is == 3. But if i should check and compare to actual naumber of items in store i will do it.

**Comment by MykhailoForge** [comment]
Closed due to opening new pr for FS-006.

**Pull request closed without merging**